risc0/zkp/src/prove/executor.rs

```diff
@@ -32,16 +32,27 @@ where
     S: CircuitStepHandler<F::Elem>,
 {
     pub circuit: &'static C,
+    // Circuit Step Handler
     pub handler: S,
+    // Control Instructions
     pub code: Vec<F::Elem>,
+    // Number of columns used for control instructions
     code_size: usize,
+    // Execution Trace Data
     pub data: Vec<F::Elem>,
+    // Number of columns used for execution trace data
     data_size: usize,
+    // Circuit output
     pub output: Vec<F::Elem>,
+    // Power of 2
     pub po2: usize,
+    // Counter for zkvm execution
```

Comment:
steps is the number of cycles (I probably should have called it num_cycles or something), i.e. it's 2^po2.  cycle is the counter.